### DISCUSSION TITLE
How to add packages when using `./reach react`?

### DISCUSSION THREAD

**Discussion by cb0806151** [reach-sh/reach-lang#74] [Q&A] [answered]
I'm currently running into this error when using `./reach react` in [this repository](https://github.com/cb0806151/pennyauctionv2)
![Screenshot 2021-03-10 181306](https://user-images.githubusercontent.com/43824792/110822203-73692100-8256-11eb-8ccc-756eecb57fed.png)
Based off of that error, is there a correct way packages should be added to a reach application that uses `./reach react`?

**Comment by cb0806151** [comment]
@DanBurton This partially relates to the discussion we had about workarounds to this issue during your office hours on Monday

**Comment by DanBurton** [comment, marked as answer]
This is not currently supported by `./reach react`, but will be supported relatively soon.
